### PR TITLE
Tag response differently if esi is available or not

### PR DESCRIPTION
**Pull request by ngilain**
[OO-FEATURE] Node cache-control policy differs with or without ESI support
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/140
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/193

### CODE REVIEW

**Comment by ngilain** [comment]
@nicolasThal : unit test added

File: FrontBundle/Tests/Manager/NodeResponseManagerTest.php [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+/**
+ * Test NodeResponseManagerTest
+ */
+class NodeResponseManagerTest extends AbstractBaseTestCase
+{
+    protected $nodeRepository;
+    protected $transverseNode;
+
+    protected $maxAge = array(
```
Comment (nicolasThal) [inline FrontBundle/Tests/Manager/NodeResponseManagerTest.php]: As both of these array are used only in the dataProvider, I would not have put them as class property

**Comment by nicolasThal** [comment]
:+1: with the array inside the dataProvider.

Nice and clean test case by the way :)

File: FrontBundle/Tests/Manager/NodeResponseManagerTest.php (hunk 1 above)
Comment (ngilain) [inline FrontBundle/Tests/Manager/NodeResponseManagerTest.php]: @nicolasThal : I did this as a 'test configuration'. That way it's easy to tweak the datas. But I can put it in the dataProvider if you prefer.

**Pull request merged**